lib/paper_trail/has_paper_trail.rb

```diff
@@ -46,6 +48,29 @@ module ClassMethods
       #   column if it exists. Default is true
       #
       def has_paper_trail(options = {})
+        # Initializing paper_trail_options with an empty hash before setting
+        # up the callback-methods is important to make them compatible with the
+        # traditional has_paper_trail method.
+        # This can move to setup_model_for_paper_trail, as soon as the
+        # cleanup_callback and setup_callbacks_from_options method is no longer
+        # needed.
+        class_attribute :paper_trail_options
+        self.paper_trail_options = {}
+
+        # TODO: Add a deprecation warning / info to use callback methods instead
+        # of the :on option?
```

Comment:
I doubt we'll be removing the `:on` option anytime soon.  Please remove comments that refer to the `:on` option being removed.